package.json

```diff
@@ -139,6 +138,7 @@
     "@digdir/designsystemet-theme": "0.15.3",
     "@material-ui/core": "4.12.4",
     "@navikt/aksel-icons": "7.14.2",
+    "@navikt/ds-icons": "^3.4.3",
```

Comment:
Maybe we should create an issue to get rid of this as well

<img width="732" alt="image" src="https://github.com/user-attachments/assets/2cd443ac-3f8a-4fa7-a987-ad21affd1da2" />